app/_hub/kong-inc/oauth2-introspection/_index.md

```diff
@@ -144,22 +154,23 @@ params:
         `X-Credential-{claim-name}`.
 ---
 
-### Flow
+## Flow
 
 ![OAuth2 Introspection Flow](/assets/images/docs/oauth2/oauth2-introspection.png)
 
-### Associate the response to a Consumer
+## Associate the response to a Consumer
```

Comment:
I wanted to say "yes, totally", but I'm also partial to simply lower-casing all "service", "route", and "consumer" instances - especially since they're lowercase in our UI now. Will go through this topic and lc instances of "Consumer".